js/views/modals/Settings/Page.js

```diff
@@ -232,7 +232,10 @@ export default class extends baseVw {
               new SimpleMessage({
```

Comment:
If you load an avatar or header that is too small, this message appears on every render. It wasn't doing this before, it was just showing when you loaded the image, which I think is the correct behavior.

With it showing on every render, every time you save the modal (whether you updated the header or now), the message will show. Also, if you close and re-open the model, it will show.